studio/gcloud_worker.py

```diff
@@ -133,8 +137,21 @@ def _get_instance_config(self, resources_needed, queue_name, timeout=300):
         # Configure the machine
         machine_type = self._generate_machine_type(resources_needed)
         self.logger.debug('Machine type = {}'.format(machine_type))
+
         with open(self.startup_script_file, 'r') as f:
             startup_script = f.read()
+        if self.runner_args is not None:
+            startup_script = startup_script.replace(
+                "{studioml_branch}", self.runner_args.branch)
```

Comment:
you can also use git_util.py to determine current branch automatically here

Reply:
Hmmm, I dont see any methods to get the branch name. Also, what if you pip installed the package from pypi?